ipxe/menu.ipxe

```diff
@@ -0,0 +1,121 @@
+#!ipxe
+
+# dhcp should be already done by boot script
+#dhcp
+
+#set variables
+set menu-timeout 30000
+
+:start
+menu iPXE boot menu for openqa.opensuse.org 
+
+item --gap --             ------------------------- Operating systems ------------------------------
+item --gap --             ------ (You will be prompted to edit cmdline after selecting an os) ------
+item --key l leapS1       (L)Boot openSUSE Leap15.3 from download.opensuse.org / ttyS1 (http)
+item --key e leapS2       (E)Boot openSUSE Leap15.3 from download.opensuse.org / ttyS2 (http)
+item --key k tumbleweed_KVM_autoyast (K)Boot openSUSE Tumbleweed KVM server from O3 repo with autoyast/ ttyS1 (http)
+item --key z tumbleweed_XEN_autoyast (Z)Boot openSUSE Tumbleweed XEN server from O3 repo with autoyast/ ttyS1 (http)
+item --key t tumbleweedS1 (T)Boot openSUSE Tumbleweed from download.opensuse.org / ttyS1 (http)
+item --key s tumbleweedS2 (S)Boot openSUSE Tumbleweed from download.opensuse.org / ttyS2 (http)
+item --key M memtest64efi (M)Memtest 64 EFI
+item --key M nbxyzbios    (B)Netboot.xyz BIOS
+item --key M nbxyzefi     (B)Netboot.xyz EFI
+item --gap --             ------------------------- Advanced options -------------------------------
+item --key c config       Configure settings
+item shell                Drop to iPXE shell 
+item reboot               Reboot computer
+item
+item --key x exit         Exit iPXE and continue BIOS boot 
+
+choose --timeout ${menu-timeout} --default exit selected || goto cancel
+set menu-timeout 0
+goto ${selected}
+
+:leapS1
+set kernel http://download.opensuse.org/distribution/leap/15.3/repo/oss/boot/x86_64/loader/linux
+set cmdline usessh=1 sshpassword=linux network=1 install=http://download.opensuse.org/distribution/leap/15.3/repo/oss/ console=ttyS1,115200n8 root=/dev/ram0 initrd=initrd textmode=1
+set initrd http://download.opensuse.org/distribution/leap/15.3/repo/oss/boot/x86_64/loader/initrd
+goto editandboot
+
+:leapS2
+set kernel http://download.opensuse.org/distribution/leap/15.3/repo/oss/boot/x86_64/loader/linux
```

Comment:
I guess you just coped that over from ariel. Maybe it makes nevertheless sense to also add entries for Leap 15.4 and 15.5 in addition. I'm not sure whether any tests rely on the order of entries here so maybe it would make sense to append these entries (instead of inserting them before Tumbleweed).